site/cypress/e2e/Cypress-Gradeable/hotkey.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('TA grading hotkey testing', () => {
+    it('toggle keyboard shortcut', () => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get('[data-testid="grade-button"]').eq(12).click();
+        cy.get('[data-testid="grading-panel-header"]').as('navigationBar');
+        cy.get('@navigationBar').type('{A}');
+        cy.get('[data-testid="autograding-results"]').should('contain', 'Autograding Testcases');
+        cy.get('@navigationBar').type('{G}');
+        cy.get('[data-testid="grading-rubric"]').should('contain', 'Grading Rubric');
+        cy.get('#edit-mode-enabled').should('not.be.checked');
+        cy.get('@navigationBar').type('{E}');
+        cy.get('#edit-mode-enabled').should('be.checked');
+        cy.get('@navigationBar').type('{O}');
+        cy.get('[data-testid="submission-browser"]').should('contain', 'Submissions and Results Browser');
+        cy.get('@navigationBar').type('{S}');
+        cy.get('[data-testid="student-info"]').should('contain', 'Student Information');
+        cy.get('@navigationBar').type('{X}');
+        cy.get('[data-testid="grade-inquiry-inner-info"]').should('contain', 'Grade Inquiry');
+        cy.get('@navigationBar').type('{T}');
+        cy.get('[data-testid="solution-ta-notes"]').should('contain', 'Solution/TA Notes');
+    });
+    it('testing discussion, peer and notebook', () => {
+        cy.login();
+        cy.visit(['sample', 'config']);
+        cy.get('[data-testid="forum-enabled"]').should('be.checked');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update']);
+        // adding the peer grading panel, discussion panel, notebook panel
+        cy.get('[data-testid="yes-discussion"]').check();
+        cy.get('[data-testid="yes-discussion"]').should('be.checked');
+        cy.get('[data-testid="page-2-nav"]').click();
+        cy.get('[data-testid="add-new-peer-component"]').click();
```

Comment:
The reason the other test is failing is because it is assuming that there is no peer component to grade for this gradeable. As an after() block could you remove the peer component? (Or create a before() statement in the test_rubric_access test to remove all peer components

As a side note, it would be good to test that adding a peer component allows students to peer grade.

Reply:
yeah it works , 
Ok i will do it for peer grade component